### PR TITLE
Add "angle" and "intersect" methods to Line3D and Plane3D!

### PR DESCRIPTION
**Pull request by marinang**

### CODE REVIEW

File: skhep/math/geometry.py [hunk 1]
```diff
@@ -608,22 +666,68 @@ def distance ( self , other ) :
         >>> print line.distance ( point )
         >>> print line.distance ( plane )        
         """
-        if   isinstance ( other , Point3D ) :
```
Comment (eduardo-rodrigues) [inline skhep/math/geometry.py]: On line 654 the example should read print(plane1!=plane2). Notice the 2 changes, the second being the most important one.

**Comment by eduardo-rodrigues** [comment]
Hi @marinang, can you deal with the comments asap? We are almost there. Thanks.

**Comment by marinang** [comment]
@eduardo-rodrigues yes sorry, I will do this either today or tomorrow. There was a Stripping deadline this week ;)

**Comment by eduardo-rodrigues** [comment]
Indeed :-).

**Comment by marinang** [comment]
@eduardo-rodrigues good to be merged?

**Comment by eduardo-rodrigues** [comment]
Indeed :-). Merging ...

**Pull request merged**